src/PowerShellEditorServices/Services/DebugAdapter/Handlers/PsesSemanticTokens.cs

```diff
@@ -0,0 +1,144 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices.Services;
+using Microsoft.PowerShell.EditorServices.Services.TextDocument;
+using System.Management.Automation.Language;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document.Proposals;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+using System;
+using Microsoft.PowerShell.EditorServices.Utility;
```

Comment:
these are usually sorted.

first, sort everything.

Then put the `System.*` ones on top:
example:
```csharp
using System;
using System.Management.Automation.Language;
using System.Threading;
using Microsoft.PowerShell.EditorServices.Services;
using Microsoft.PowerShell.EditorServices.Services.TextDocument;
using Omnisharp.Extensions.LanguageServer.Protocol;
```